docs/en/ingest-management/elastic-agent/running-on-kubernetes-standalone.asciidoc

```diff
@@ -259,6 +259,70 @@ the following configuration should be added as an extra input in the DaemonSet m
 Refer to <<kubernetes-provider,dynamic inputs and Kubernetes provider>> for more information
 about shaping dynamic inputs for autodiscovery.
 
+[discrete]
+== Autodiscover targeted Pods using hints
+
+In order to achieve dynamic monitoring of Pods/containers, one can use hints. This can be achieved by properly
+enabling hints for the `kubernetes` dynamic provider. For more information about this feature refer to
+<<kubernetes-provider,dynamic inputs and Kubernetes provider>>.
+
+In the standalone policy ensure that the feature is enabled:
+
+In order to enable hints one need to add the `hints.enabled: true` in the provider's configuration, ie:
+
+[source,yaml]
+----
+providers:
+  kubernetes:
+    node: "kind-control-plane"
+    hints.enabled: true
+----
+
+Then ensure that the proper volumes and volumeMounts are added properly. These sections are already defined in the
+provided kubernetes manifest, so it's only required to uncomment these sections.
+
+In order to enable hints one need to add the `hints.enabled: true` in the provider's configuration, ie:
+
+[source,yaml]
+----
+providers:
+  kubernetes:
+    node: "kind-control-plane"
+    hints.enabled: true
+----
+
+In order to enable hints one need to add the `hints.enabled: true` in the provider's configuration, ie:
+
+[source,yaml]
+----
+volumeMounts:
+- name: external-inputs
+  mountPath: /etc/elastic-agent/inputs.d
+...
+volumes:
+- name: external-inputs
+  emptyDir: {}
+...
+----
+
+Along with these sections an init container is required which is responsible for downloading the hints' templates.
+The init container is already defined, so uncomment the respective section:
+
+[source,yaml]
+----
+initContainers:
+- name: k8s-templates-downloader
+  image: busybox:1.28
+  command: ['sh']
+  args:
+    - -c
+    - >-
+      mkdir -p /etc/elastic-agent/inputs.d &&
+      wget -O - https://github.com/elastic/elastic-agent/archive/8.5.0.tar.gz | tar xz -C /etc/elastic-agent/inputs.d --strip=5 "elastic-agent-main/deploy/kubernetes/elastic-agent-standalone/templates.d"
```

Comment:
You are right. We properly handle it in the official manifests, see https://github.com/elastic/elastic-agent/blob/main/deploy/kubernetes/elastic-agent-standalone/elastic-agent-standalone-daemonset.yaml#L37. Here since it's just docs' example it should be fine but I will make it to use the proper version dynamically like what we do in the `curl` command in the very beginning of this guide.